### PR TITLE
Announcements screen

### LINKED ISSUE #21: As a user, I want to be able to write announcements so that I can find teammates

### PR DESCRIPTION
**Pull request by ktotam1**

### CODE REVIEW

**Review by DerouichAbdessalam — DISMISSED** [review]
Greaat work ! only tests left

**Comment by hamzaremmal** [comment]
Tests are missing here

**Review by hamzaremmal — DISMISSED** [review]
Lets now work on the tests ;-)

File: app/src/main/java/com/github/versus/sports/Sport.java
Comment (hamzaremmal) [inline app/src/main/java/com/github/versus/sports/Sport.java:1]: You have deleted this file

File: app/src/androidTest/java/com/github/versus/FsPostManagerTests.java [hunk 1]
```diff
@@ -48,7 +48,7 @@ public void CorrectFsPostInsert_Get() throws ExecutionException, InterruptedExce
 
             // Creating a test post
             Post post = new Post( "_test_", new Timestamp(2023, Month.AUGUST, 18, 11, 15, Timestamp.Meridiem.AM) ,
-                    new Location("tirane", 0, 0), new ArrayList<>(), 15, Sport.FOOTBALL);
```
Comment (hamzaremmal) [inline app/src/androidTest/java/com/github/versus/FsPostManagerTests.java:51]: [:-)](https://en.wikipedia.org/wiki/Ball_(association_football))

**Review by hamzaremmal — APPROVED** [review]
lets get this merge

**Pull request merged**